docs/DafnyRef/UserGuide.md

```diff
@@ -156,9 +156,9 @@ exit code.
 
 ## 24.8. Verification
 
-In this section, we suggest a methodology to figure out [why a single assertion might not hold](#sec-verification-debugging), we explain how to [make the verifier more focused on particular assertions](#sec-assertion-batches) to devote more time to prove them, and we also give some more examples of [useful options and attributes to control verification](#sec-command-line-options-and-attributes-for-verification).
+In this section, we suggest a methodology to figure out [why a single assertion might not hold](#sec-verification-debugging), we propose methodologies to deal with [assertions that slow a proof down](#sec-verification-debugging-slow), we explain how to [verify assertions in parallel or in a focused way](#sec-assertion-batches) to devote more time to prove them, and we also give some more examples of [useful options and attributes to control verification](#sec-command-line-options-and-attributes-for-verification).
```

Comment:
Why does verifying assertions in parallel devote more time to prove them?

Reply:
methodologies → techniques
(A methodology (singular) is already a set of techniques, so methodologies are sets of techniques)

Reply:
>Why does verifying assertions in parallel devote more time to prove them?

It devotes more time if you sum all the times spent by each core used by the verifier. But I'm removing this remark as it's not very relevant.

> methodologies → techniques

Reworded. Note that I think that although a "methodology" is a "set of techniques", it's a subtype of it, because it feels like the techniques are homogeneous in some way. That's why I'll keep "methodology" for the first part, and "techniques" for the second.